src/KafkaW/Consumer.cpp

```diff
@@ -129,13 +130,29 @@ bool Consumer::topicPresent(const std::string &TopicName) {
 
 void Consumer::updateMetadata() {
```

Comment:
Now this function is doing something more complicated it would be nice to have a doxygen comment to explain.